nodebuilder/core/tls.go

```diff
@@ -0,0 +1,58 @@
+package core
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/celestiaorg/celestia-node/libs/utils"
+)
+
+const (
+	cert   = "cert.pem"
+	key    = "key.pem"
+	xtoken = "xtoken.json"
+)
+
+func EmptyTLSConfig() *tls.Config {
+	return &tls.Config{MinVersion: tls.VersionTLS12}
+}
```

Comment:
nit: don't need to hide details into func, would be easier to read if value used in place

Reply:
The idea was to keep the dependency inside core package and does not bloat all other places